test/features/opening_hours/data/repositories/opening_hours_repository_impl_test.dart

```diff
@@ -70,10 +47,15 @@ void main() {
         const Right(
           OpeningHours(
             allOpeningHours: {
+              1: 'Closed',
```

Comment:
Use `DateTime.monday` etc instead of ints

```suggestion
               DateTime.monday: 'Closed',
```